examples/sdk-app-example/src/main/java/com.bugsnag.android/example/ExampleActivity.kt

```diff
@@ -139,13 +146,14 @@ class ExampleActivity : AppCompatActivity() {
      * When sending a handled error, a callback can be registered, which allows the Error Report
      * to be modified before it is sent.
      */
+    @Suppress("UNUSED_PARAMETER")
     fun crashWithCallback(view: View) {
         val e = RuntimeException("Customized Error Report")
 
-        Bugsnag.notify(e, Callback { report ->
+        Bugsnag.notify(e) { report ->
             // modify the report
-            report.error?.metaData = generateUserMetaData()
-        })
+            report.error.metaData = generateUserMetaData()
```

Comment:
Did the nullability of `Report.error` change?

Reply:
Yes, in a recent release of bugsnag-android. Updating the example app's dependency means we can get rid of the unnecessary null check.